core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -84,7 +84,7 @@
   private static final CarbonProperties CARBONPROPERTIESINSTANCE = new CarbonProperties();
 
   /**
-   * porpeties .
+   * porpeties
```

Comment:
porpeties should be Properties, can you describe it in detail?

Reply:
Right, this spell error has corrected